201-vm-efficientip-vhd/azuredeploy.parameters.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
format the JSON

Reply:
Ok. I am looking at the JSON in notepad and VS.  I don't see anything wrong with the format.  Please explain what you would like me to correct?

Reply:
in VS Code - do SHIFT+ALT+F

Reply:
Thanks removed the tabs